website/docs/docs/get-started/getting-started/getting-set-up/setting-up-snowflake.md

```diff
@@ -184,7 +184,7 @@ There are two ways to connect dbt Cloud and Snowflake. The first option is Partn
 <Lightbox src="/img/snowflake_tutorial/snowflake_classic_ui_partner_connect.png" title="Snowflake Classic UI - Partner Connect" />
 </p>
 
-  * **Snowflake New UI:** If you’re using the new web interface, you’ll want to click on your name in the upper left hand corner and then click on Partner Connect in the drop down menu.  You can scroll down to find the dbt tile, or search for dbt in the search bar and it will float to the top. Click on the tile to open up the connect box.
+  * **[Snowflake New UI](https://docs.snowflake.com/en/user-guide/ecosystem-partner-connect.html):** If you’re using the new web interface, click on the home icon in the upper left-hand corner and then click on Admin title, and then Partner Connect.  You can scroll down to find the dbt tile, or search for dbt in the search bar and it will float to the top. Click on the tile to open up the connect box.
```

Comment:
```suggestion
  * **[Snowflake New UI](https://docs.snowflake.com/en/user-guide/ecosystem-partner-connect.html):** If you’re using the new web interface, click on the home icon in the upper left-hand corner and then click on **Admin** title, and then **Partner Connect**.  You can scroll down to find the dbt tile, or search for dbt in the search bar and it will float to the top. Click on the tile to open up the connect box.
```